.travis.yml

```diff
@@ -52,42 +52,59 @@ install:
   - pip install --upgrade zkpython
   - go get gopkg.in/ini.v1 gopkg.in/tylerb/graceful.v1
   - go get golang.org/x/sys/unix
+  - sudo bash -c "echo '/tmp/core.%p.%E' > /proc/sys/kernel/core_pattern"
+  - mkdir /tmp/oio
+  - git fetch --tags
+  - git submodule update --init --recursive
 
 env:
-  - TEST_SUITE=sdk
-  - TEST_SUITE=3copies,with-service-id
-  - TEST_SUITE=ec,with-random-service-id
-  - TEST_SUITE=rebuilder,with-service-id
-  - TEST_SUITE=ec
-  - TEST_SUITE=repli,go-rawx,fsync
-  - TEST_SUITE=3copies
-  - TEST_SUITE=multi-beanstalk
-  - TEST_SUITE=small-cache,fsync
-  - TEST_SUITE=webhook
-  - TEST_SUITE=slave
-  - TEST_SUITE=cli
-  - TEST_SUITE=worm
-  - TEST_SUITE=build,unit,copyright,variables
-  - TEST_SUITE=mover,with-service-id
+  global:
+  - CMAKE_OPTS="-DENABLE_CODECOVERAGE=on -DCMAKE_INSTALL_PREFIX=/tmp/oio -DLD_LIBDIR=lib -DZK_LIBDIR=/usr/lib -DZK_INCDIR=/usr/include/zookeeper -DAPACHE2_LIBDIR=/usr/lib/apache2 -DAPACHE2_INCDIR=/usr/include/apache2 -DAPACHE2_MODDIR=/tmp/oio/lib/apache2/module"
+  - G_DEBUG=fatal_warnings
+  - G_SLICE=always-malloc
```

Comment:
`G_DEBUG=fatal_warnings` is fine, but we may accelerate several tests if we stop using `G_SLICE=always-malloc`.